### ISSUE TITLE
[SCREENREADER]: Pittsburgh Sidenav - Sidenav headings lack semantic structure 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3441]
## Description
**[SCREENREADER]: Pittsburgh Sidenav - Sidenav headings lack semantic structure **
The "headings" in sidenav are coded as ~~divs~~ buttons without action, lack semantic structure. [WCAG 2.4.6](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html)

## Type of Issue
Template

### From November 18, 2019
![Screen Shot 2019-11-18 at 4.15.10 PM.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/e83b1ef9-6fda-4015-8205-e789d628f5e1)

#### From November 13, 2019
![pittsburgh-sidenav-headings-divs.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/aa276417-df1d-4079-ba00-18fc85a9f826)

## Point of Contact

Jennifer

## Acceptance Criteria

As a  screen reader user, I want to hear the content structure with headings to navigate the page content so I may understand the presented content. 

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all

## Steps to Recreate

1. Enter `https://staging.va.gov/pittsburgh-health-care/` in browser
2. Start screenreading device listed in Environment
3. Listen for the content structure

## Possible Fixes (optional)

A possible semantic structure that may be helpful. Alternatively, another container element with headings would be possible as long as the heading structure of the page complied with 508 guidelines.

### Preferred solution:

```diff
<nav aria-label="secondary">
  <dl>
  <dt role="heading">Services and Locations</dt>
    <dd><a href="#">Health services</a></dd>
    <dd><a href="#locations-list">Locations</a>
      <ul id="locations-list">
        <li>List items</li>
        <li>List items</li>
      </ul>
    </dd>
  <dt role="heading">News and Events</dt>
+    <dd><a href="#" class="sample-on-this-page" aria-current="page">Events</a></dd>
    <dd><a href="#">Stories</a></dd>
    <dd><a href="#">News releases</a></dd>
  <dt role="heading">About VA Pittsburgh</dt>
    <dd><a href="#about-pitt">About us</a>
      <ul id="about-pitt">
        <li>List items</li>
        <li>List items</li>
      </ul>
    </dd>
    <dd><a href="#">And so on&hellip;</a></dd>
  </dl>
</nav>
```

### another option

```HTML
<dl id="sidenav">
  <dt>Services and Locations</dt>
    <dd><a href="#">Health services</a></dd>
    <dd><a href="#locations">Locations</a>
      <ul id="locations">
        <li><a href="#">University Drive campus</a></li>
        <li>And so on&hellip;</li>
      </ul>
    </dd>
  <dt>News and Events</dt>
    <dd><a href="#">Events</a></dd>
    <dd><a href="#">Stories</a></dd>
    <dd><a href="#">News releases</a></dd>
  <dt>About VA Pittsburgh</dt>
    <dd><a href="#about-us">About us</a>
      <ul id="about-us">
        <li><a href="#">And so on&hellip;</a></li>
    </dd>
</dl>
```

### An alternate take:

```HTML
<aside>
  <h2>Services and Locations</h2>
  <ul>
    <li><a href="#">Health services</a></li>
    <li><a href="#locations-list">Locations</a>
      <ul id="locations-list">
        <li>List items</li>
        <li>List items</li>
      </ul>
    </li>
  </ul>
  <h2>News and Events</h2>
  <ul>
    <li><a href="#">Events</a></li>
    <li><a href="#">Stories</a></li>
    <li><a href="#">News releases</a></li>
  </ul>
  <h2>About VA Pittsburgh</h2>
  <ul>
    <li><a href="#about-pitt">About us</a>
      <ul id="about-pitt">
        <li>List items</li>
        <li>List items</li>
      </ul>
    </li>
        <li><a href="#">And so on&hellip;</a></li>
  </ul>
</aside>
```

**Comment by mmiddaugh** [comment]
This issue should be resolved as part of the design changes in [#7212](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7212). Will confirm once changes are available in staging.  Moving this to the Icebox until then.

**Comment by mmiddaugh** [comment]
This is ready for validation at your convenience, @jenstrickland

**Comment by jenstrickland** [comment]
Validated. 🎉
